README.txt

```diff
@@ -1,3 +1,7 @@
+ This version of pjsip has been forked off r4448, found at
+ http://svn.pjsip.org/repos/pjproject/trunk/, and modified slightly by
+ Metaswitch.
```

Comment:
Do we need to specify a time period here?

Reply:
It's listed elsewhere (52 times!), so I plan to leave it out here 😄

Reply:
Fair enough 😺